### DISCUSSION TITLE
Re-implement N3 & SPARQL Lexers for documentation

### DISCUSSION THREAD

**Discussion by nicholascar** [RDFLib/rdflib#1577] [Documentation]
Previously, the readthedocs rdflib documentation used two Pygments lexers that for N3 & SPARQL that are now no longer available:

https://github.com/gjhiggins/sparql_pygments_lexer.git#egg=SPARQL_Pygments_Lexer
https://github.com/gjhiggins/n3_pygments_lexer.git#egg=Notation3_Pygments_Lexer

We need to replace these with available lexers. One, SPARQL, at least is already available, as is Turtle:

https://pygments.org/docs/lexers/#lexers-for-semantic-web-and-rdf-query-languages-and-markup

**Comment by ghost** [comment]
> Previously, the readthedocs rdflib documentation used two Pygments lexers that for N3 & SPARQL that are now no longer available:

Mea culpa. 

> sparql_pygments_lexer.git n3_pygments_lexer.git
> 
> We need to replace these with available lexers. One, SPARQL, at least is already available, as is Turtle:
> 
> https://pygments.org/docs/lexers/#lexers-for-semantic-web-and-rdf-query-languages-and-markup

More sensible to use the Pygments supported-and-maintained versions.

I cloned the n3 lexer from https://github.com/gniezen/n3pygments
